test/Misc/serialized-diagnostics-batch-mode-primary-errors.swift

```diff
@@ -1,18 +0,0 @@
-// Ensure that an error in a primary causes an error in the errorless primary.
```

Comment:
This test is still relevant, isn't it?

Reply:
I see, it is partly still relevant because it tests an erroneous primary. I'll modify it and put it back. Thanks for the catch! (Although we may have other tests that cover this.)